hswidget/selectpalettewidget/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package selectpalettewidget contains palette select widget
+// used in dcc, dc6 and dt1 editors
+package selectpalettewidget
```

Comment:
sorry for moving that in here, it was generating `import cycle not allowed` error